common/workunit/package.cpp

```diff
@@ -17,6 +17,7 @@
 
 
 #include "platform.h"
+#include "jlib.hpp"
```

Comment:
You seem to have changed quite a few files to add this include, but with no other changes - was wondering why?